cachito/workers/nexus_scripts/pip_after_content_staged.groovy

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Can you clarify why we do not need to (or cannot) block outbound connections the same way we do for NPM?

Reply:
Sure! I included an explanation in this comment in the header of the file